manifests/testing/rook-ceph/operator.yaml

```diff
@@ -143,7 +143,7 @@ spec:
         #   value: "role=storage-node; storage=rook, ceph"
         # Allow rook to create multiple file systems. Note: This is considered
         # an experimental feature in Ceph as described at
-        # http://docs.ceph.com/docs/master/cephfs/experimental-features/#multiple-filesystems-within-a-ceph-cluster
+        # http://docs.ceph.com/docs/latest/cephfs/experimental-features/#multiple-filesystems-within-a-ceph-cluster
```

Comment:
Looks like it's actually https://docs.ceph.com/en/latest/cephfs/experimental-features/#multiple-filesystems-within-a-ceph-cluster